modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -255,21 +255,52 @@ class GKernelType<K, std::function<R(Args...)> >
                         public G_ID_HELPER_CLASS(Class)
 // {body} is to be defined by user
 
+#define GET_HELPER(_1, _2, _3, _4, _5, _6, NAME, ...) NAME
+#define API(...) __VA_ARGS__
```

Comment:
these names are too generic (especially `API`). I'd rename it to something more specific to avoild hardly-debuggable collisions

Reply:
Fixed